templates/records/verification.hbs

```diff
@@ -0,0 +1,34 @@
+{{#>default section=sections.records page=sections.records.subsections.verification}}
+<div style="min-height: 60vh;">
+    {{! This is the starter for a section page
+        -------------------------------------------------------
+        - The tag at the top brings in the default portions
+        - Data comes in from JSON files, please put all data/strings in the JSON files
+        - Keep the data organized, if you're adding financial data put it under "finances"
+    }}
+    <div class="col-md-12">
+        <h4><b>Proof of Enrollment</b></h4>
+        <table class="table table-striped">
```

Comment:
Does this need to be a table? Maybe one of these? http://getbootstrap.com/components/#panels-tables